frontend/src/app/utils/format.ts

```diff
@@ -0,0 +1,5 @@
+import {formatCurrency} from '@angular/common';
+
+export function formatCZK(value: number, currencySymbol = 'Kč'): string {
+  return formatCurrency(value, 'cs-CZ', currencySymbol, 'symbol', '1.0-0');
```

Comment:
I would prefer `.toLocalString()`, since it uses user's settings. And we already use it in the app.

Reply:
Maybe, there's an issue with decimal points to solve. So just check it, pls.

Reply:
Good point, changed. (Decimal point was not problem)